### PR TITLE
Add addwiki/wikibase-api and Wikibase EntityStore to components page

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
@addshore shame on you - `addwiki/wikibase-api` is the only thing with a red badge on the page now!

red

File: source/components.html [hunk 1]
```diff
@@ -140,6 +149,15 @@
     </tr>
     </thead>
     <tr>
+        <td><a href="https://github.com/wmde/AskSerialization">addwiki/wikibase-api</a></td>
```
Comment (Tpt) [inline source/components.html]: URL issue

Comment (JeroenDeDauw) [inline source/components.html]: Good catch

**Comment by addshore** [comment]
@JeroenDeDauw I'll work on that when I find a moment! :)

**Pull request merged**